### PR TITLE
Basic code for e2e build tests for encryption

### PR DESCRIPTION
**Pull request by gvallee**
**Description of the Pull Request (PR):**

Add e2e tests for building encrypted containers. At the moment, only for the `build` command with a pass phrase.
We currently have an issue with getting error messages, which will be fixed separately. At the moment, we implement a work around that basically deactivate checking the error message.

**This fixes or addresses the following GitHub issues:**

- Addresses #4253 

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

**Review by WestleyK — COMMENTED** [review]
Is this still a WIP?

**Comment by gvallee** [comment]
> Is this still a WIP?

Yes, this is based on the new code from #4271. As soon as #4271 is merged, I will update, ensure that everything is working and remove the WIP label.

**Comment by gvallee** [comment]
@WestleyK it is now ready for review

File: e2e/imgbuild/imgbuild.go [hunk 1]
```diff
@@ -731,6 +737,100 @@ func (c *imgBuildTests) buildDefinition(t *testing.T) {
...
+
+	if !strings.Contains(string(out), "cryptsetup 2.") {
+		return fmt.Errorf("incompatible cryptsetup version")
+	}
+
+	return nil
+}
+
+// buildEncryptPassphrase is exercising the build command for encrypted containers
+// while using a passphrase. Not that it covers both the normal case and when the
```
Comment (ikaneshiro) [inline e2e/imgbuild/imgbuild.go]:
```suggestion
// while using a passphrase. Note that it covers both the normal case and when the
```

**Review by ikaneshiro — APPROVED** [review]
Lgtm, small comment fix

**Pull request merged**